File: КОМПОНЕНТЫ/OPartner/Examples/CBuildr5/XlDemo2.cpp
//---------------------------------------------------------------------------
#include <vcl.h>
#pragma hdrstop

#include "XlDemo2.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "OpDbOfc"
#pragma link "OpModels"
#pragma link "OpShared"
#pragma resource "*.dfm"
TDataModule1 *DataModule1;
//---------------------------------------------------------------------------
__fastcall TDataModule1::TDataModule1(TComponent* Owner)
    : TDataModule(Owner)
{
}
//---------------------------------------------------------------------------

File: КОМПОНЕНТЫ/OPartner/Examples/Delphi/xldemo2.pas
unit XLDemo2;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  OpModels, OpDbOfc, Db, DBTables, OpShared;

type
  TDataModule1 = class(TDataModule)
    tblCustomers: TTable;
    qryChartData: TQuery;
    dsCustomers: TDataSource;
    tblOrders: TTable;
    dsOrders: TDataSource;
    mdlCustomers: TOpDataSetModel;
    mdlOrders: TOpDataSetModel;
    mdlChartData: TOpDataSetModel;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  DataModule1: TDataModule1;

implementation

uses XLDemo1;

{$R *.DFM}

end.
